fabric/Makefile

```diff
@@ -75,24 +76,28 @@ clean-patch:
 $(FABRIC_TLCC):
 	ln -sfn $$(/bin/pwd)/../tlcc $(FABRIC_TLCC)
 
+$(FABRIC_VALIDATION):
+	ln -sfn $$(/bin/pwd)/../validation/fpc $(FABRIC_VALIDATION)
+
```

Comment:
I am wondering if we really need this.

Reply:
I hope not. that would kindof defeat a bit the purpose of build tags?

Reply:
I did the same... and for a reason.
The validator has to access the Go protobufs, which "we" have in the go path, while "Fabric" has then in the vendor folder.
So, without this link, go build complains that it wants the protobufs in Fabric's vendor folder (following a similar path: `github.com/hyperledger/...`). With this link, instead, go build grabs the protobufs from vendor and it's happy. In this case, we might think of dropping our FPC dependency on protobufs (though TLCC might still require them).

Reply:
i doubt that the symlinks will be acceptable: what would happen if we also vendor protos in fpc?  That said, wouldn't vendoring become obsolete once you move to go modules. Actually, fabric-protos-go _is_ modularized and fabric does include fabric-protos-go in go.mod. I think the version in vendor is (should be) obsolete!

Reply:
as discussed with @g2flyer we keep the symlinks until we bump to fabric v2.2